platforms/defaults_windows.go

```diff
@@ -78,14 +81,21 @@ func revision(v string) int {
 	return r
 }
 
+func prefix(v string) string {
+	parts := strings.Split(v, ".")
+	if len(parts) < 4 {
+		return v
+	}
+	return strings.Join(parts[0:2], ".")
+}
+
 // Default returns the current platform's default platform specification.
 func Default() MatchComparer {
-	major, minor, build := windows.RtlGetNtVersionNumbers()
-	return matchComparer{
-		defaults: Ordered(DefaultSpec(), specs.Platform{
-			OS:           "linux",
-			Architecture: runtime.GOARCH,
-		}),
-		osVersionPrefix: fmt.Sprintf("%d.%d.%d", major, minor, build),
+	// Windows also supports linux via LCOW
+	linux := specs.Platform{
+		OS: "linux",
+		Architecture: "amd64",
```

Comment:
LCOW wont match x86?

Reply:
woops I should have had `runtime.GOARCH` https://github.com/containerd/containerd/blob/604c462d7bc59f967511c923e21c86014a4d0891/platforms/defaults_windows.go#L87

based on https://github.com/containerd/containerd/blob/604c462d7bc59f967511c923e21c86014a4d0891/platforms/defaults_windows.go#L87 sounds like we will remove this anyways